wp-content/themes/access/src/js/modules/office-map.js

```diff
@@ -131,7 +131,7 @@ class OfficeMap {
     this._Autocomplete.addListener('place_changed', () => {
       const place = this._Autocomplete.getPlace();
 
-      if (place) {
+      if (place.geometry.location) {
```

Comment:
I think to make sure it doesn't cause any issues, taking an additive approach to the condition might be best;

`if (place && place.geometry.location) {`

Reply:
@theasteve I led you astray;

`if (place.geometry)` is correct.